src/client/types.ts

```diff
@@ -82,7 +82,6 @@ export type LaunchServerOptions = {
     password?: string
   },
   downloadsPath?: string,
```

Comment:
Do we still need a separate path for downloads or we could just merge it with artifactsPath?

Reply:
We'll probably move `downloadsPath` to `newContext` as well. Let's see whether videos work out as we want them to.